### ISSUE TITLE
unsupported operand type(s) for *=: 'float' and 'NoneType'

### ISSUE THREAD

**Issue by naniqingkuang** [tensorflow/tensorflow#18593]
any one happened this?

tf_X = tf.placeholder(tf.float32, [None, 8, 8, 1])
tf_Y = tf.placeholder(tf.float32, [None, 10])

conv_out2 = tf.nn.conv2d(relu_feature_maps1, conv_filter_w2, strides=[1, 2, 2, 1], padding='SAME',use_cudnn_on_gpu=False) + conv_filter_b2

print("log+++")
print (type(conv_out2.get_shape()[0].value))

batch_mean, batch_var = tf.nn.moments(conv_out2, axes=[0, 1, 2])

**Comment by reedwm** [comment]
Please fill out the issues template and give a small self-contained code example reproducing the issue. The above code is not self contained because there are some undefined variables like `relu_feature_maps1`.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 35 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
